### ISSUE TITLE
Getting "Error creating formatter: Nagios::NagiosFormatter (RuntimeError)" using cucumber > 0.3.95

### ISSUE THREAD

**Issue by jeremypruitt** [auxesis/cucumber-nagios#2]
I had to make sure I had cucumber 0.3.95 installed or I would get a NagiosFormatter error. Here's the full text:

```
Cucumber::Ast::Visitor is deprecated and will be removed. You no longer need to inherit from this class.
Error creating formatter: Nagios::NagiosFormatter (RuntimeError)
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/ast/visitor.rb:7:in `initialize'
./features/support/nagios.rb:4:in `initialize'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/configuration.rb:146:in `new'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/configuration.rb:146:in `formatters'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/configuration.rb:131:in `map'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/configuration.rb:131:in `formatters'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/configuration.rb:62:in `build_runner'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/main.rb:53:in `execute!'
/usr/lib/ruby/gems/1.8/gems/cucumber-0.3.104/lib/cucumber/cli/main.rb:24:in `execute' bin/cucumber:15
```

**Comment by jeremypruitt** [comment]
Either a change to the Rakefile to make sure the cucumber version isn't above 0.3.95 or modify the code to accommodate new versions of cucumber.

**Comment by auxesis** [comment]
Yeah, this is another lazy dependencies problem. Right now the easiest fix is to lock Cucumber to 0.3.95, like you mentioned. 

We're going to have to bump the Cucumber dependency version at some stage, so i'll have a look at upgrading the formatter. 

Thanks for reporting this!
